package.json

```diff
@@ -375,6 +409,11 @@
                 "title": "Open Dashboard",
                 "category": "Kubernetes"
             },
+            {
+                "command": "extension.vsKubernetesCopy",
+                "title": "Copy",
```

Comment:
Should this perhaps be Copy Resource ID (or whatever k8s calls that kind/name string)?  Copy implies I am copying the resource itself...

Reply:
In k8s schema, it called `Name`. So changed it to `Copy Name`